src/Extension/Resources/config/task/AcpAppAsyncApiCodeGenerateTask.yaml

```diff
@@ -18,11 +18,3 @@ placeholders:
       option: "asyncapi-file"
       description: "Path to AsyncAPI file"
       type: string
-  - name: "%organization%"
```

Comment:
do not forget about task version

Reply:
Not needed to increase the version. it's an optional placeholder and not used in command